packages/core/src/testLint/eslint.test.ts

```diff
@@ -15,16 +15,32 @@ describe('eslint', function () {
                 '../../node_modules/.bin/eslint',
                 '-c',
                 '../../.eslintrc.js',
+                // Note: eslint currently does not support multiple  --ignore-path args.
+                // Use --ignore-pattern as a workaround.
                 '--ignore-path',
-                '../../.eslintignore',
+                '../../.gitignore',
+                '--ignore-pattern',
+                '**/*.json',
+                '--ignore-pattern',
+                '**/*.gen.ts',
+                '--ignore-pattern',
+                '**/types/*.d.ts',
+                '--ignore-pattern',
+                '**/src/testFixtures/**',
+                '--ignore-pattern',
+                '**/resources/js/graphStateMachine.js',
                 '--ext',
                 '.ts',
-                '.',
+                '../amazonq',
+                '../core',
+                '../toolkit',
+                // TODO: fix lint issues in scripts/
+                // '../../scripts',
```

Comment:
run eslint on all packages. 

Future: `testLint/` should live outside of core/